packages/terra-consumer-nav/src/components/nav-items/NavItem.scss

```diff
@@ -19,8 +19,7 @@
     display: block;
     text-decoration: none;
 
-    &:hover,
-    &:focus {
+    &:hover {
       background: rgba(255, 255, 255, 0.4);
```

Comment:
Do these need to be theme-able?

Reply:
probably not a bad idea to be, i will update the hover effects to be theme-able